File: cfgs/java_cfgs_foo/FIND_THE_MAXIMUM_SUBARRAY_XOR_IN_A_GIVEN_ARRAY.java
Nodes
1;special;;;
2;statement;int ans = Integer.MIN_VALUE;14;
3;statement;int i = 0;15;
4;conditional;foo_1 < foo_2;16;
5;statement;int curr_xor = 0;18;
6;statement;int j = foo_1;19;
7;conditional;foo_1 < foo_2;20;
8;statement;curr_xor = foo_1 ^ foo_2[foo_3];22;
9;statement;ans = fun_1(foo_1, foo_2);23;
10;statement;j++;21;
11;statement;i++;17;
12;statement;return foo_1;26;
13;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;12;
4;6;
5;7;
6;8;11;
7;9;
8;10;
9;7;
10;4;
11;13;
12;

File: cfgs/pythoncfgs/FIND_THE_MAXIMUM_SUBARRAY_XOR_IN_A_GIVEN_ARRAY.py
Nodes
1;statement;foo_1 = -2147483648;8;
2;conditional;fun_1(foo_1);9;
3;statement;return foo_1;14;
4;statement;foo_1 = 0;10;
5;conditional;foo < foo;11;
6;statement;foo_1 = foo_2 ^ foo_3[foo_4];12;
7;statement;foo_1 = fun_1(foo_2, foo_3);13;
8;exit;;
Edges
0;2;
1;3;4;
2;8;
3;5;
4;2;6;
5;7;
6;5;
7;